<html><head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head
    content must come *after* these tags -->
    <meta name="description" content="">
    <meta name="author" content="">
    <title>Bin Chen Lab</title>
    <!-- Bootstrap core CSS -->
    <link href="dist/css/bootstrap.min.css" rel="stylesheet">
    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
    <link href="assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">
    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
    <!--[if lt IE 9]>
      <script src="../../assets/js/ie8-responsive-file-warning.js"></script>
    <![endif]-->
    <script src="assets/js/ie-emulation-modes-warning.js"></script>
    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements
    and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
    <!-- Custom styles for this template -->
    <link href="carousel.css" rel="stylesheet">
  </head><body>
    <nav class="navbar navbar-inverse navbar-fixed-top">
      <div class="container">
        <div class="navbar-header">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand" href="#">Bin Chen Lab @ MSU</a>
        </div>
        <div id="navbar" class="collapse navbar-collapse">
          <ul class="nav navbar-nav">
            <li class="active">
              <a href="./index.html">Home</a>
            </li>
            <li>
              <a href="./research.html">Research</a>
            </li>
            <li>
              <a href="./team.html">Team</a>
            </li>
            <li>
              <a href="./publication.html">Publications</a>
            </li>
            <li>
              <a href="./resource.html">Resources</a>
            </li>
            <li>
              <a href="./contact.html">Contact</a>
            </li>
            <li>
              <a href="./join_us.html">Join us</a>
            </li>
          </ul>
        </div>
        <!--/.nav-collapse -->
      </div>
    </nav>
    <!-- /.carousel -->
    <!-- Marketing messaging and featurettes==================================================-
    ->
    <!-- Wrap the rest of the page in another container to center all the content. -->
    <!-- /.row -->
    <!-- START THE FEATURETTES -->
    <div class="container marketing">
      <hr class="featurette-divider">
      <div class="row featurette">
        <div class="col-md-8 col-md-push-3">
          <p class="lead">A systems approach to discover new therapeutics</p>
          <p>Rapidly decreasing costs of molecular measurement technologies not only enable profiling of disease sample molecular features (e.g., transcriptome, proteome, metabolome) at different levels (e.g., tissues, single cells), but also enable measuring of molecular signatures of individual drugs in clinically relevant models. Exploring methods to relate diseases to potentially efficacious drugs through various molecular features is critically important in the discovery of new therapeutics. The target-based drug discovery approach that focuses on interfering with individual targets is challenged by the lack of drug efficacy, drug resistance, and off-target effects. We propose to employ a systems-based approach that identifies drugs that reverse the molecular state of a disease. Using this approach, we have successfully identified drug candidates for three cancers: Ewing’s sarcoma (Pessetto, Oncotarget, 2016), liver cancer (Chen, Gastroenterology, 2017) and basal cell carcinoma in the past few years. In the Ewing sarcoma work, this systems-approach achieved a hit rate of >50% in predicting effective drugs.  In the liver cancer work, we demonstrated that the expression of disease genes was reversed in a clinically relevant mouse model after drug treatment. In our recent pan-cancer analysis, we found that the potency to reverse cancer gene expression correlates to drug efficacy (Chen, Nature Communications, 2017). Moving forward, we will move our drug candidates to the clinic for the liver cancer patients, meanwhile, we will explore the following topics.</p>
            <p><b>1. Identify drugs to overcome drug resistance</b>. Our past focus was on primary cancers, which have a large amount of molecular data publicly available. Since drug resistance is emerging in various cancers and relevant data are growing rapidly, we are excited to apply the systems approach to study drug resistance. For example, in neoadjuvant chemotherapy (NAC), breast cancer patients who achieve pathologic complete response (pCR) likely have long-term survival, yet there remains a significant number of patients with poor response to NAC and who fail to achieve pCR. New drugs are therefore needed to overcome chemoresistance to help patients achieve pCR. We propose to identify drugs that reverse the signature of non-responders. </p>
            <p><b>2. Discover novel oncogene inhibitors</b>. Despite the discovery of numerous driver oncogenes, many of the most prevalent oncogenic alterations cannot be blocked with small molecule inhibitors. We will use the systems-based approach to identify small molecules that reverse oncogene specific signatures. We are working on c-MYC and several others.</p>
            <p><b>3. Develop the Open Cancer Therapeutic Discovery portal (OCTAD)</b> To help computational biologists contribute to the therapeutic discovery pipeline improvement, we have released our <a href="https://github.com/Bin-Chen-Lab/octad_desktop" target="_blank">desktop version</a>. To help researchers easily use our pipeline to predict drug candidates for the disease of their interest, we are actively develop a <a href="https://octad.org" target="_blank">web portal</a>. 
          </div>
        <div class="col-md-3 col-md-pull-8">
          <img class="featurette-image img-responsive center-block" data-src="" alt="Generic placeholder image" src="fig/repositioning.jpg">
        </div>
      </div>
      <hr class="featurette-divider">
      <div class="row featurette">
        <div class="col-md-8 col-md-push-3">
          <p class="lead">Personalized cancer therapy</p>
          <p>Current preclinical and clinical approaches select therapies primarily
            based on actionable mutations, yet patients may have no actionable mutations
            or multiple actionable mutations that are hard to prioritize, suggesting
            the need for additional biomarkers. The recent disappointing result from
            the SHIVA trial indicates that more predictive biomarkers of drug efficacy
            are needed to select therapies. In addition to mutations, other molecular
            data (e.g., gene, protein) have been widely explored as biomarkers in personalized
            therapy, including gene expression based biomarkers such as Oncotype DX
            and MammaPrint in breast cancer treatment, however, the biomarkers for
            many cancer drugs are currently quite limited as it takes many years to
            run clinical trials to identify and validate a biomarker for a single drug.
            The recent efforts have enabled the large-scale identification of various
            types of molecular biomarkers through correlating drug sensitivity with
            molecular profiles of pre-treatment basal cancer cell lines. We are interested
            in developing methods to match these biomarkers to individual patients
            to inform therapy in the clinic. Our first effort is centered on building statistical models to improve drug sensitivity data (Zhiyue Hu, 2019, PSB). </p>
        </div>
        <div class="col-md-3 col-md-pull-8">
          <img class="featurette-image img-responsive center-block" data-src="" alt="Generic placeholder image" src="fig/prescription.gif">
        </div>
      </div>
      <hr class="featurette-divider">
      <div class="row featurette">
        <div class="col-md-8 col-md-push-3">
          <p class="lead">A big data approach to study metastatic cancer</p>
          <p>Metastasis is the most common cause of cancer-related death, as such, there is an urgent need to discover new therapies to treat metastasized cancers. We are extending our primary cancer work to study metastatic cancer. We are interested to use open big data to ask significant questions related to metastatic cancer research. One recent example is to <a href="https://doi.org/10.1101/337287" target="_blank">evaluate cell lines and organoids as models for metastatic cancer</a>. </p>
        </div>
        <div class="col-md-3 col-md-pull-8">
          <img class="featurette-image img-responsive center-block" data-src="" alt="Generic placeholder image" src="fig/metastatic_cancer.png" height="60%" width="60%">
        </div>
      </div>
        
      <hr class="featurette-divider">
      <div class="row featurette">

        <div class="col-md-8 col-md-push-3">
          <p class="lead">Deep learning and Deep reinforcement learning in drug discovery</p>
          <p>The new technologies in deep learning and deep reinforcement learning are simply blowing our mind everyday. We are interested in implementing these technologies into drug discovery. Examples include using deep learning autoencoder to learn gene expression profiles (William Zeng, BMC Genomics, 2018) and exploring GAN to design new compounds.  Too much to do! If you understand this figure, please join us:)</p>
        </div>
         <div class="col-md-3 col-md-pull-8">
          <img class="featurette-image img-responsive center-block" data-src="" alt="Generic placeholder image" src="fig/deep_learning_fig.png">
        </div>          
      </div>
        
      <hr class="featurette-divider">
        <div class="col-md-8 col-md-push-3">
          <p class="lead">Drug mechanisms in a cellular context</p>
          <p>One challenge in the systems-based therapeutic discovery approach is that
            drug mechanisms are barely known. We will leverage drug-target prediction
            models we previously developed (
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/22859915" target="_blank">1</a>,
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/22360769" target="_blank">2</a>,
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/21903625 " target="_blank">3</a>) and the new drug/shRNA-mediated gene expression profiles to infer mechanisms
            of actions in individual cell lines.</p>
        </div>

      <div class="row featurette">
         <div class="col-md-3 col-md-pull-8">
          <img class="featurette-image img-responsive center-block" data-src="" alt="Generic placeholder image" src="fig/moa.jpg">
        </div>
      </div>
        
      <hr class="featurette-divider">
      <div class="row featurette">
        <div class="col-md-10 col-md-push-0">
          <p class="lead">Past projects</p>
          <p>We used Semantic Web Technologies to integrate (
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/20478034 " target="_blank">Chem2Bio2RDF</a>), annotate (
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/22401035" target="_blank">Chem2Bio2OWL</a>), and mine (
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/22859915" target="_blank">SLAP</a>) open linked data for drug discovery.</p>
        </div>
      </div>
      <hr class="featurette-divider">
      <footer>
        <p></p>
            <p>© 2016 Bin Chen Lab,
              <a href="http://www.phd.msu.edu/" target="_blank">Department of Pediatrics and Human Development</a> and
              <a href="https://www.phmtox.msu.edu/" target="_blank">Department of Pharmacology and Toxicology</a>
              <a href="http://humanmedicine.msu.edu/" target="_blank"> College of Human Medicine</a>,
              <a href="https://www.msu.edu/" target="_blank">Michigan State University</a>
            </p>
      </footer>
      <div class="row">
        <!-- /.container -->
        <!-- Bootstrap core JavaScript==================================================-
        ->
    <!-- Placed at the end of the document so the pages load faster -->
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
        <script>
          window.jQuery || document.write('<script src="assets/js/vendor/jquery.min.js"><\/script>')
        </script>
        <script src="dist/js/bootstrap.min.js"></script>
        <!-- Just to make our placeholder images work. Don't actually
        copy the next line! -->
        <script src="assets/js/vendor/holder.min.js"></script>
        <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
        <script src="assets/js/ie10-viewport-bug-workaround.js"></script>
      </div>
    </div>
  

</body></html>
